### PR TITLE
speeds up get_hearers_in_view/getallcontents just a little bit

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

image

image

## Why It's Good For The Game

i got bored and did a thing

## Changelog
ehh no real changes

### CODE REVIEW

**Comment by kevinz000** [comment]
(someone take a good stare at this make sure i'm not doing anything too stupid it's the middle of the night)

**Comment by vuonojenmustaturska** [comment]
`COMSIG_ATOM_HEARER_IN_VIEW` 🤔

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -199,11 +199,9 @@
 /proc/get_hearers_in_view(R, atom/source)
 	// Returns a list of hearers in view(R) from source (ignoring luminosity). Used in saycode.
 	var/turf/T = get_turf(source)
-	. = list()
```
Comment (AnturK) [inline code/__HELPERS/game.dm]: You're changing it to return null instead of empty list by default, nothing cares about this ?

Comment (kevinz000) [reply code/__HELPERS/game.dm]: oh
haha wait
fuck

**Comment by spookydonut** [comment]
jesus thats like such a tiny improvement

**Comment by vuonojenmustaturska** [comment]
get_hearers_in_view is iirc a top 10 proc on the live server

**Pull request merged**